datadog/tests/cassettes/TestAccDatadogMetricTagConfiguration_Error.yaml

```diff
@@ -0,0 +1,23 @@
+---
+version: 1
+interactions:
+- request:
+    body: |
+      {"data":{"attributes":{"aggregations":[{"space":"avg","time":"avg"},{"space":"sum","time":"sum"}],"metric_type":"gauge","tags":[]},"id":"tf_TestAccDatadogMetricTagConfiguration_Error_local_1618930539","type":"manage_tags"}}
+    form: {}
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+    url: https://api.datadoghq.com/api/v2/metrics/tf_TestAccDatadogMetricTagConfiguration_Error_local_1618930539/tags
+    method: POST
+  response:
+    body: |
+      {"status":"error","code":403,"errors":["Forbidden"],"statuspage":"http://status.datadoghq.com","twitter":"http://twitter.com/datadogops","email":"support@datadoghq.com"}
```

Comment:
Not sure why we're getting a 403 forbidden here?

Reply:
I had issues with cassettes in the past, I think we should probably reach out to `#intg-tools-libs`

Reply:
This was a tricky one to figure out, turns out this 403 was probably from when you were implementing the new field. The test itself doesn't actually do any api calls, since all the errors are caught by terraform itself, so there was no new cassette to overwrite the original 403 recording.